rasa/core/processor.py

```diff
@@ -312,6 +327,16 @@ def _get_action(self, action_name):
                 message.text, parse_data["intent"], parse_data["entities"]
             )
         )
+        # check if we pick up intents that aren't in the domain
```

Comment:
Please extract this also into a separate function. Same as you did for entities.